src/libNode/Node.h

```diff
@@ -420,7 +419,7 @@ class Node : public Executable, public Broadcastable
     void StartSynchronization();
 
     /// Performs PoW mining and submission for DirectoryService committee membership.
-    bool StartPoW(const uint64_t& block_num, uint8_t dsDifficulty,
+    bool StartPoW(const uint64_t& block_num, uint8_t ds_difficulty,
```

Comment:
Although it is automated, I prefer it to be without  ```_``` for consistency with other code. same for block_num.